gensim/models/word2vec.py

```diff
@@ -71,6 +71,7 @@
 import logging
 import sys
 import os
+import warnings
```

Comment:
`logger.warn()` messages don't appear in the console window right? Does the user have to check into the log file to see the warning?